src/css/custom.css

```diff
@@ -93,6 +95,24 @@
   font-display: swap;
 }
 
```

Comment:
should cover all supported font weights

Reply:
As Zee asked to support only font weight of 400 and change all font weights of brb page to 400..so supported only 400 weight